moto_sync_from_s3/Tiltfile

```
@@ -20,6 +20,7 @@ def moto_sync_from_s3(
     sync_on_start = False,
     resource_deps = ["motoserver"],
     labels = ["motoserver"],
+    resource_name = None
```

```suggestion
    resource_name = "",
```
Starlark can handle empty strings as defaults. Also appended a trailing slash here.